roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml

```diff
@@ -48,37 +48,9 @@
     kind: node
     state: absent
     labels: "{{ glusterfs_nodeselector | lib_utils_oo_dict_to_list_of_dict }}"
-  with_items: "{{ groups.all }}"
+  with_items: "{{ groups.oo_all_hosts }}"
```

Comment:
Probably should be groups.oo_nodes_to_config because some hosts might not be nodes.

Reply:
Fait enough.